### ISSUE TITLE
Update to latest version on Open VSX

### ISSUE THREAD

**Issue by agichim** [michaelgmcd/vscode-language-babel#80]
On Visual Studio Marketplace it's v `0.0.30`

On Open VSX it's v `0.0.29` [Link here](https://open-vsx.org/extension/mgmcdermott/vscode-language-babel)

Thanks!

**Comment by michaelgmcd** [comment]
I'm unclear what Open VSX is. How do I update it there? I use vsce to publish.

**Comment by agichim** [comment]
Open VSX is the extensions marketplace for VSCodium. I'm unsure who publishes the extensions on the Open VSX marketpkace, I'd imagine the developer does?

**Comment by medihack** [comment]
Would be nice to have this extension in Open VSX (which for example is used by [Gitpod](https://gitpod.io)).

**Comment by lrkwz** [comment]
> I'm unclear what Open VSX is. How do I update it there? I use vsce to publish.

Open CSV is the markeplace adapter used by the open version of vscode named [vscodium](https://vscodium.com/).

Follow instructions in https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by lrkwz** [comment]
> On Visual Studio Marketplace it's v `0.0.30`
> 
> On Open VSX it's v `0.0.29` [Link here](https://open-vsx.org/extension/mgmcdermott/vscode-language-babel)
> 
> Thanks!

BTW the extension isn't there anymore

> Extension Not Found: mgmcdermott.vscode-language-babel

**Comment by michaelgmcd** [comment]
Working on this now, waiting on https://github.com/EclipseFdn/open-vsx.org/issues/627
